pkgs/applications/networking/dyndns/dyndnsc/default.nix

```
@@ -1,62 +1,50 @@
 {
   lib,
-  stdenv,
   python3Packages,
-  fetchPypi,
+  fetchFromGitHub,
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "dyndnsc";
-  version = "0.6.1";
+  version = "develop-2024-11-04";
```

```suggestion
  version = "0.6.1-unstable-2024-11-04";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning